### PR TITLE
only trigger workflow on push to dev

### PR DESCRIPTION
**Pull request by kateharwood**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -4,11 +4,10 @@ name: CI
 
 # Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the dev branch
+  # Triggers the workflow on push to the dev branch
   push:
     branches: [ dev ]
   pull_request:
-    branches: [ dev ]
```
Comment (tildechris) [inline .github/workflows/main.yml:11]: Can you think of a sanity check / unit test that we can still do here?

Comment (tildechris) [reply .github/workflows/main.yml:11]: We don't need to consider it in the scope of this PR, but some sort of check would be good.  Perhaps https://github.com/jimhester/lintr with check syntax checking to start?

**Review by tildechris — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (kateharwood) [reply .github/workflows/main.yml:11]: Sounds good, I will add an issue for that.

Comment (sgratzl) [reply .github/workflows/main.yml:11]: see also https://devtools.r-lib.org/reference/check.html

**Pull request merged**